### PR TITLE
remake and update prepare_deploy_cfg

### PR DESCRIPTION
**Pull request by Tianhao-Gu**
remake and copy/paste current prepare_deploy_cfg

### CODE REVIEW

**Review by bio-boris — APPROVED** [review]
Looks good! Did you copy and paste these all from somewhere?

File: lib/src/us/kbase/fbatools/PredictAuxotrophyParams.java [hunk 1]
```diff
@@ -2,6 +2,7 @@
 package us.kbase.fbatools;
```
Comment (bio-boris) [inline lib/src/us/kbase/fbatools/PredictAuxotrophyParams.java:2]: Where did you get these changes from?

File: lib/src/us/kbase/fbatools/ModelObjectSelectionParams.java [hunk 2]
```diff
@@ -78,6 +81,21 @@ public ModelObjectSelectionParams withSaveToShock(Long saveToShock) {
         return this;
     }
 
+    @JsonProperty("fulldb")
```
Comment (bio-boris) [inline lib/src/us/kbase/fbatools/ModelObjectSelectionParams.java:84]: What is this and where did it come from?

File: lib/src/us/kbase/fbatools/PredictAuxotrophyParams.java (hunk 1 above)
Comment (Tianhao-Gu) [reply lib/src/us/kbase/fbatools/PredictAuxotrophyParams.java:2]: @bio-boris that was just run `make` from latest sdk

File: lib/src/us/kbase/fbatools/ModelObjectSelectionParams.java (hunk 2 above)
Comment (Tianhao-Gu) [reply lib/src/us/kbase/fbatools/ModelObjectSelectionParams.java:84]: @bio-boris re`make` too.

**Comment by Tianhao-Gu** [comment]
@bio-boris prepare_deploy_cfg.py was copied and pasted from another module which was made from latest sdk.

**Comment by Tianhao-Gu** [comment]
created PR in cshenry/fba_tools instead.

**Pull request closed without merging**